docs/updates/2019-02.md

```diff
@@ -0,0 +1,93 @@
+# February 2019
+
+Welcome to the February 2019 Qlik Core update. We're highlighting a couple of important changes and
+improvements which we hope you will like.
+
+## Usability and design changes in catwalk
+
+Since being moved to stable last month, we've made several small usability improvements and design changes along with a
+couple of bug fixes.
+
+### QlikView select excluded coloring
+
+We have added support for the *select excluded* selection state, which can be used for deeper data model analysis. A
+value enters the selected excluded state because the value was first selected, and then excluded by a selection in
+another field. This value will be described in catwalk with *green on light grey background*.
+
+!!! Note
+    Please note the color scheme follows QlikView coloring and corresponds to *dark grey and check mark* in Qlik
+    Sense.
+
+More information about Qlik's selection model can be found in the
+[Qlik Sense help pages](https://help.qlik.com/en-US/sense/Subsystems/Hub/Content/Sense_Hub/Selections/associative-selection-model.htm).
+
```

Comment:
Does the guide describe the selection model itself, or from a sense perspective? If from a sense perspective, remove the reference. To avoid confusion.

Reply:
I think this is a border line case. The guide is extensive and good to read but are written from a Qlik Sense perspective. I think it is ok with the previous info regarding the checkmark. I think it is a bigger undertaking to describe the selection model from a Qlik Core perspective and we miss information if we don't add it.